databallpy/load_data/tracking_data/tracab.py

```diff
@@ -0,0 +1,264 @@
+from typing import Tuple
+
+import bs4
+import numpy as np
+import pandas as pd
+from bs4 import BeautifulSoup
+from tqdm import tqdm
+
+from databallpy.load_data.metadata import Metadata
+
+
+def load_tracking_data_tracab(
```

Comment:
For opta it is called `load_opta_event_data`, so maybe call this one `load_tracab_tracking_data` for consistancy?